deploy/README.md

```diff
@@ -1,15 +1,19 @@
 # Deployment Guide
 
-This page has instructions for collecting Kubernetes logs, metrics, and events; enriching them with deployment, pod, and service level metadata; and sending them to Sumo Logic. See our [documentation guide](https://help.sumologic.com/Solutions/Kubernetes_Solution) for details on our Kubernetes Solution.
+This page has instructions for collecting Kubernetes logs, metrics, and events;
+enriching them with deployment, pod, and service level metadata; and sending them to Sumo Logic.
+See our [documentation guide](https://help.sumologic.com/Solutions/Kubernetes_Solution)
+for details on our Kubernetes Solution.
 
 - [Deployment Guide](#deployment-guide)
   - [Solution overview](#solution-overview)
   - [Minimum Requirements](#minimum-requirements)
   - [Support Matrix](#support-matrix)
   - [Installation with Helm](./docs/Installation_with_Helm.md)
-  - [Non Helm Installation](./docs/Non_Helm_Installation.md) 
+  - [Non Helm Installation](./docs/Non_Helm_Installation.md)
   - [Adding Additional FluentD Plugins](./docs/Additional_Fluentd_Plugins.md)
   - [Advanced Configuration/Best Practices](./docs/Best_Practices.md)
+  - [Authenticating with docker registry](./docs/Authenticating_with_docker_registry.md)
```

Comment:
I'd call it either `Docker Hub` or `container registry` with the preference for the latter.

Reply:
```suggestion
  - [Authenticating with container registry](./docs/Authenticating_with_docker_registry.md)
```